vault/resource_aws_secret_backend.go

```diff
@@ -98,6 +98,13 @@ func awsSecretBackendResource() *schema.Resource {
 				Computed:    true,
 				Description: "Template describing how dynamic usernames are generated.",
 			},
+			"local": {
+				Type:        schema.TypeBool,
+				ForceNew:    true,
+				Optional:    true,
+				Default:     false,
```

Comment:
Suggest making this `Computed` instead, which lets TF know that this state can be read and set to state from Vault. Ideally we like to let Vault handle all the default values it can :)

Reply:
Are Computed and Default mutually exclusive? I was trying to make sure that it would default to false, since this would be a net new flag, and changing to true requires a recreate on the resource

Reply:
It sounds like `Computed` will effectively pull this value from Vault's default (which is false) if the optional value is not provided.

Reply:
Never mind, since this is a `ForceNew` and we don't want to recreate existing TF pipelines, it's fine to leave this as is